crates/orderbook/openapi.yml

```diff
@@ -508,9 +508,9 @@ components:
       description: |
         How good should the price estimate be?
 
-        **NOTE**: Orders are supposed to be created from `optimal` price estimates.
+        **NOTE**: Orders are supposed to be created from `verified` price estimates.
       type: string
-      enum: [fast, optimal]
+      enum: [fast, optimal, verified]
```

Comment:
It would be nice if we explained what the values mean. Someone looking at these docs won't understand the difference. Not required for this PR.

Reply:
Good point. Added description.